stylesheets/partials/components/_buttons.scss

```diff
@@ -30,4 +30,9 @@ button {
 	  	background: lighten($gold, 20);
 	  	color: $dark-gray;
 	  }
+}
+input[type=button][disabled], 
+input[type=submit][disabled], 
+button[disabled] {
+    color: #999;
```

Comment:
1. This color should be in a variable in stylesheets/modules/colors.scss, so we don't have to look up the color when we find something needing to be disabled.
1. For the fun of it, I applied the style to one of our normal yellow buttons, and it looks weird. Something should probably happen to the button background as well.